reactioncommerce/reaction-component-library
# Skip company when validating filled fields

As described Here: https://github.com/reactioncommerce/reaction-component-library/issues/416

Resolves #416
Impact: **|major**  
Type: **bugfix**





## Component
Nothing relevant here.

## Screenshots
Not a new or visually altered component.

## Testing
1. Add a product and go to checkout in the storefront
2. Add a shipping address
3. Reload the page
4. Notice that the shipping address can be updated properly now.
